### PR TITLE
[#157183717] User can submit an incident report

### PR DESCRIPTION
**Pull request by nelmandela**
### What does this PR do?

  •Allows users to submit an incident report

### Description of task to be completed:

  •Add an AssetIncidentReport model
  •Make migrations
  •Add test

### How can this be manually tested?

  • *python manage.py test*
  • `/admin/core/assetincidentreport/`

### Screenshot
incident2
incident
menu

### CODE REVIEW

**Comment by iankigen** [comment]
@egichuri @alexkiura @andela-Taiwo Kindly review

File: core/models/asset.py [hunk 1]
```diff
@@ -247,11 +255,32 @@ def save(self, *args, **kwargs):
         super(AssetCondition, self).save(*args, **kwargs)
 
 
+class AssetIncidentReport(models.Model):
```
Comment (egichuri) [inline core/models/asset.py]: i think we might need to use `TextField ` for some of these fields.. attaching the form used on the app.

**Comment by egichuri** [comment]
screen shot 2018-05-28 at 17 35 20

File: core/models/asset.py (hunk 1 above)
Comment (iankigen) [reply core/models/asset.py]: working on it

**Pull request merged**